bin/fallback-module-directory-webpack-plugin.js

```diff
@@ -0,0 +1,140 @@
+/*eslint-env node*/
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+
+// Note, this has some inspiration from the AliasPlugin and its implementation
+// @see https://github.com/webpack/enhanced-resolve/blob/v4.1.0/lib/AliasPlugin.js
+
+function startsWith( string, searchString ) {
```

Comment:
I know it comes from `AliasPlugin.js`, but couldn't we just use native `String.startsWith()` instead of this function?

Reply:
ha!  Yup we could, I just forgot about the native implementation.

Reply:
Fixed in 2ad64ff